### PR TITLE
(titus)  Setting interestingHealthProvider (if present) as a titus job label

### PR DESCRIPTION
**Pull request by jeyrschabu**
- This label will be used for discovery purposes

### CODE REVIEW

**Comment by jeyrschabu** [comment]
- Grabbing interestingProviderNames from the request and explicitly set it as a label
- interestingProviderNames: "Titus,..." (comma-separated for lists) is how the user should set it. Orca will extract that correctly as a list

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/description/TitusDeployDescription.groovy [hunk 1]
```diff
@@ -44,6 +44,7 @@ class TitusDeployDescription extends AbstractTitusCredentialsDescription impleme
   int retries
   int runtimeLimitSecs
   Boolean useApplicationDefaultSecurityGroup = true
+  String interestingHealthProviderNames
```
Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/description/TitusDeployDescription.groovy]: Isn't this a list?

Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/description/TitusDeployDescription.groovy]: nvm, saw the comment above. LGTM

Comment (jeyrschabu) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/description/TitusDeployDescription.groovy]: It's a comma-separated String and is processed like shown in here:  https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplierSpec.groovy#L77

**Pull request merged**